services/diddoc/queries/base_handler.go

```diff
@@ -20,13 +19,7 @@ type BaseQueryHandler struct {
 
 func (b *BaseQueryHandler) SetNext(c services.ResolverContext, next BaseQueryHandlerI) error {
 	// All the query handlers are dereferencing by default
-	acceptHeader := c.Request().Header.Get(echo.HeaderAccept)
```

Comment:
Removing this check here, for error responses in each handler for W3DIDRES is going to return a W3DIDURL